.github/workflows/reusable_multi_numa.yml

```diff
@@ -15,11 +15,11 @@ jobs:
   multi_numa:
     name: "${{matrix.os}}, ${{matrix.build_type}}, shared=${{matrix.shared_library}}"
     # run only on upstream; forks will not have the HW
-    if: github.repository == 'oneapi-src/unified-memory-framework'
+    #if: github.repository == 'oneapi-src/unified-memory-framework'
```

Comment:
Why have you commented it out?

Reply:
ok, commented for testing purpose

Reply:
Please do not resolve issues on your own. The issue is always resolved by the submitter.

Reply:
> ok, commented for testing purpose

If it is just for testing then you should:
1) convert this PR to draft (since it is not read yet)
2) use a temporary patch like https://github.com/ldorau/unified-memory-framework/commit/a590c27dfe8fdb98930da37dc29dbb158487edff on my `DEBUG_CI_run_only_multi_numa` branch to test only one CI job that you are changing and do not occupy other resources, when you do not need them (the resources in public UMF CI are very limited).